static/app/views/starfish/utils/useSpansQuery.tsx

```diff
@@ -83,7 +85,7 @@ export function useWrappedDiscoverTimeseriesQuery<T>({
     },
     DiscoverQueryProps
   >({
-    route: 'events-stats',
+    route: overriddenRoute ?? 'events-stats',
```

Comment:
We want to allow overriding instead of cloning all these hooks since `traces-stats` is a direct sub-set of `event-stats` for now.